tachyon/zk/plonk/circuit/examples/simple_lookup_circuit.h

```diff
@@ -38,7 +38,7 @@ class SimpleLookupConfig {
   void Load(Layouter<F>* layouter) const {
     layouter->AssignLookupTable(
         absl::Substitute("$0-bit table", Bits), [this](LookupTable<F>& table) {
-          for (size_t row = 0; row < size_t{1} << Bits; ++row) {
+          for (RowIndex row = 0; row < RowIndex{1} << Bits; ++row) {
```

Comment:
Could you please import row_index.h for `RowIndex`?

Reply:
I intend to not include any of header other than circuit.h. For user's convenience, I want this header to be the only header to be included.